src/open_inwoner/conf/production.py

```diff
@@ -29,6 +29,12 @@
 else:
     DIGID_ENABLED = False
 
+EHERKENNING_MOCK = config("EHERKENNING_MOCK", default=False)
+if not DEBUG:
+    AUTHENTICATION_BACKENDS += ["eherkenning.backends.eHerkenningBackend"]
+elif EHERKENNING_MOCK:
+    AUTHENTICATION_BACKENDS += ["eherkenning.mock.backends.eHerkenningBackend"]
```

Comment:
This means that `DEBUG` could be true in production? I would expect that perhaps in `conf.base` we conditionally add `eherkenning.mock.backends.eHerkenningBackend`, depending on the value of `DEBUG` and `EHERKENNING_MOCK`, but `production.py` simply has `AUTHENTICATION_BACKENDS += ["eherkenning.backends.eHerkenningBackend"]`. Perhaps I misunderstand how the settings are used.

Reply:
I'd recommend to do this the same as how this is done with the DigiD backends

Reply:
Discussed with Alex: I'll remove the check for `DEBUG` and simply check if `EHERKENNING_MOCK` is true